cypress/integration/login.js

```diff
@@ -21,7 +21,8 @@ describe('Sourcebooks login', function() {
         cy.get('.page__title').contains('Timesheets')
         cy.get('.calendar').should('be.visible')
         cy.get('.tile.form').should('be.visible')
-        cy.get('.user-info__title').contains('Demo User');
+        cy.get('.user-info__title').contains('Egle Bucevice');
         cy.get('.main-nav').find('li').should('have.length', 1);
+        cy.get('.calendar__day.calendar--today.calendar--selected').contains('24');

```

Comment:
This locator is too long and this might not be working all the time as ".calendar--selected" is not always today as if you select another day it will fail